synapse/handlers/presence.py

```diff
@@ -2129,6 +2135,8 @@ def handle_update(
         is_mine: Whether the user is ours
         wheel_timer
         now: Time now in ms
+        override: True if this state should persist until another update occurs.
```

Comment:
Nitpicking, but I wonder if this arg should be called `persist` or similar?

(I am guessing the presence updates normally expire after a certain amount of time, and that this flag prevents that expiry?)

Reply:
> (I am guessing the presence updates normally expire after a certain amount of time, and that this flag prevents that expiry?)

Precisely.